pkg/volume/util/types/types.go

```diff
@@ -25,6 +25,10 @@ import (
 	"k8s.io/mount-utils"
 )
 
+var (
+	NodeExpansionNotRequired = "volume.kubernetes.io/node-expansion-not-required"
```

Comment:
Who is going to set the annotation and when?

Reply:
https://github.com/kubernetes-csi/external-resizer/pull/496